### PR TITLE
[Docs] Add Volumes and Nodes

### PR DESCRIPTION
**Pull request by SSmallMonster**
#### What this PR does / why we need it:
Add Volumes and Nodes
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by SSmallMonster** [comment]
cc~ @windsonsea

File: docs/docs/volumes_and_nodes/create_hwameistor_volumes/disk_volumes.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+sidebar_position: 2
+sidebar_label: "Disk Volume"
+---
+
+# Disk Volume
+
+Another type of data volume provided by HwameiStor is the raw disk data volume. 
+This type of data volume is based on the raw disk on the node and is directly mounted for container use. 
+Therefore, this type of data volume provides more efficient data read and write performance, 
+fully releasing the performance of the disk.
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/create_hwameistor_volumes/disk_volumes.md:11]:
```suggestion
HwameiStor provides another type of data volume known as raw disk data volume.
This volume is based on the raw disk present on the node and can be directly mounted for container use.
As a result, this type of data volume offers more efficient data read and write performance,
thereby fully unleashing the performance of the disk.
```

File: docs/docs/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 1
+sidebar_label: "LVM Volume"
+---
+
+# LVM Volume
+
+HwameiStor provides LVM-based data volumes. 
+This type of data volume provides read and write performance close to that of native disks, 
+and also offers advanced features such as data volume expansion, migration, high availability, and more.
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md:10]:
```suggestion
HwameiStor provides LVM-based data volumes,
which offer read and write performance comparable to that of native disks.
These data volumes also provide advanced features such as data volume expansion, migration, high availability, and more.
```

File: docs/docs/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+The following example will demonstrate how to create a simple non-HA type data volume.
+
+## Steps
+
+### 1. Prepare LVM Storage Node
+
+It is necessary to ensure that the storage node has available capacity. If there is not enough capacity,
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md:18]:
```suggestion
Ensure that the storage node has sufficient available capacity. If there is not enough capacity, 
```

File: docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+# Disk Storage Node
+
+Raw disk storage nodes provide applications with raw disk data volumes and 
+maintain the mapping between raw disks and raw disk data volumes on the storage node.
+
+## Steps
+
+### 1. Prepare Disk Storage Node
+
+Add the node into the Kubernetes cluster, or select a Kubernetes node.
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md:15]:
```suggestion
Add the node to the Kubernetes cluster or select a Kubernetes node.
```

File: docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+$ kubectl -n hwameistor get pod -o wide | grep k8s-worker-2
+hwameistor-local-disk-manager-sfsf1     2/2     Running   0     19h   10.6.128.150      k8s-worker-2   <none>  <none>
+
+# check LocalDiskNode resource
+$ kubectl get localdisknode k8s-worker-2
+NAME           FREECAPACITY   TOTALCAPACITY   TOTALDISK   STATUS   AGE
+k8s-worker-2                                              Ready    21d
+```
+
+## Add the storage node into HwameiStor
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md:42]:
```suggestion
### 2. Add the storage node into HwameiStor
```

File: docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+Construct the storage pool of the node by adding a LocalStorageClaim CR as below:
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md:44]:
```suggestion
Create the storage pool of the node by adding a LocalStorageClaim CR as below:
```

File: docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+  name: k8s-worker-2
+spec:
+  nodeName: k8s-worker-2
+  owner: local-disk-manager
+  description:
+    diskType: SSD
+EOF
+```
+
+## Post check
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md:60]:
```suggestion
### 3. Post check
```

File: docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+Finally, check if the node has constructed the storage pool by checking the LocalDiskNode CR.
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/disk_nodes.md:62]:
```suggestion
Finally, check if the node has created the storage pool by checking the LocalDiskNode CR.
```

File: docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+sidebar_label: "LVM Storage Node"
+---
+
+# LVM Storage Node
+
+LVM storage nodes pool the disks on the node and provide LVM-type data volumes for applications.
+
+## Steps
+
+### 1. Prepare Storage Node
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md:12]:
```suggestion
### 1. Prepare a storage node
```

File: docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+Add the node into the Kubernetes cluster, or select a Kubernetes node.
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md:14]:
```suggestion
Add the node into the Kubernetes cluster or select a Kubernetes node.
```

File: docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+The node should have all the required items described in [Prerequisites](../../quick_start/install/prereq.md).
+
+For example, the new node and disk information are as follows:
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md:17]:
```suggestion
For example, suppose you have a new node with the following information:
```

File: docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+  name: k8s-worker-4
+spec:
+  nodeName: k8s-worker-4
+  owner: local-storage
+  description:
+    diskType: SSD
+EOF
+```
+
+## Post check
```
Comment (windsonsea) [inline docs/docs/volumes_and_nodes/node_expansion/lvm_nodes.md:63]:
```suggestion
### 3. Post check
```

File: docs/i18n/cn/docusaurus-plugin-content-docs/current/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+provisioner: lvm.hwameistor.io
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
+allowVolumeExpansion: true
+EOF 
+```
+
+### 3. 创建数据卷 PVC
+
+使用以下命令创建一个名称为 `hwameistor-lvm-volume` 的 PVC
```
Comment (windsonsea) [inline docs/i18n/cn/docusaurus-plugin-content-docs/current/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md:53]:
```suggestion
使用以下命令创建一个名称为 `hwameistor-lvm-volume` 的 PVC：
```

File: docs/i18n/cn/docusaurus-plugin-content-docs/current/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: hwameistor-storage-lvm-ssd
+EOF
+```
+
+### 4. 使用数据卷
+
+使用以下命令创建一个 `nginx` 应用并使用数据卷 `hwameistor-lvm-volume`
```
Comment (windsonsea) [inline docs/i18n/cn/docusaurus-plugin-content-docs/current/volumes_and_nodes/create_hwameistor_volumes/lvm_volumes.md:73]:
```suggestion
使用以下命令创建一个 `nginx` 应用并使用数据卷 `hwameistor-lvm-volume`：
```

**Review by windsonsea — COMMENTED** [review]
Pick some consistency issues.

**Comment by SSmallMonster** [comment]
@windsonsea Thanks, all resolved.

**Comment by windsonsea** [comment]
Thanks. LGTM.

**Pull request merged**